Ontology/Asset/Equipment/Meter/GasMeter.json

```diff
@@ -12,16 +12,12 @@
       "writable": true
     },
     {
-      "@type": [
-        "MassFlowRate",
-        "Property"
-      ],
+      "@type": "Property",
```

Comment:
Are we removing the semantic type? Reason?

Reply:
I've filtered out a couple of semantic types (but not all) on request from Rick, since these are in REC in SI units, which may not be suitable for Imperial customers. I'd be more than happy to revert this and add semantic types all over, but those would be SI, so I understand it is a bit of a hard sell.